spec/adapters/alma_adapter/bib_spec.rb

```diff
@@ -71,61 +135,115 @@
     end
   end
 
-  describe "records with no availability" do
+  describe "record with no physical inventory" do
     it "doesn't have an AVA tag" do
-      # find an alma record with no 952 from the publishing job to add it as a fixture.
+      expect(described_class.get_bib_record(unsuppressed_no_ava)['AVA']).to be nil
     end
   end
 
-  describe "with an alma record that has an ARK" do
-    it "exposes the ark" do
-      # find an alma record with an ark.princeton.edu
-    end
-  end
-
-  describe "alma record with no item" do
-    # it has a holding
-    # it doesn't have an item. This should be checked on the Holding
-    it "has a holding" do
+  describe "record with electronic inventory" do
+    it "has an AVE tag" do
+      expect(described_class.get_bib_record(unsuppressed_no_ava)['AVE']).not_to be nil
     end
   end
 
-  describe "alma holding with order information" do
-    # alma record with a po line.
-    it "displays ..." do
+  describe ".get_items_for_bib" do
+    it "returns a Hash" do
+      expect(described_class.get_items_for_bib(unsuppressed)).to be_a Hash
     end
   end
 
   # no need to check for a 959 in Alma. This will be a check after the index
-  describe "alma holding with order information" do
+  describe "A record with order information" do
```

Comment:
Can we make this and many of the below tests a "context" within `describe ".get_items_for_bib"`? This will help group them as tests for that method.

Reply:
yes I will do this thanks @hackmastera